SwrveSDK/src/google/java/com/swrve/sdk/Swrve.java

```diff
@@ -229,4 +212,24 @@ protected boolean checkPlayStoreSpecificArguments(String purchaseData, String re
     public void processIntent(Intent intent) {
         SwrveLogger.e(LOG_TAG, "The processIntent method is Deprecated and should not be used anymore");
     }
+
+    @Override
+    public void setRegistrationId(String regId) {
```

Comment:
Really really minor but can you move this method up higher and place it just after the getRegistrationId. Makes its nicer to read